DependencyInjection/Configuration.php

```diff
@@ -30,7 +27,7 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->booleanNode('autoinclude')->defaultTrue()->end()
-                ->scalarNode('base_path')->defaultValue('bundles/norzechowiczaceeditor/ace')->end()
+                ->scalarNode('base_path')->defaultValue('vendor/ace')->end()
```

Comment:
No pb.

I can't write `web/vendor/ace` because  `$extension->getAssetUrl($this->basePath.'/'.$this->mode.'/'.$file.'.js');` will then return 
`/web/vendor/ace/....js` and we'll need a subfolder `web` in `web` -_-

Anyway i'll make it clearer on the README